index.js

```diff
@@ -50,26 +72,60 @@ getUsersCreditDetails(usersArray);
 
 const genderView = users => {
   // Your code goes here ...
+  
+  let femaleUser = users
+    .filter(function(user) {
+      return user.gender == 'female';
+    })
+    .map(function(user) {
+      return user.firstName + ' ' + user.lastName;
+    })
+  let maleUser = users
+    .filter(function(user) {
+      return user.gender == 'male';
+    })
+    .map(function(user) {
+      return user.firstName + ' ' + user.lastName;
+    })
+
+  let obj = {maleUser, femaleUser}
+  return obj;
+  
 };
 
-genderView(usersArray);
-// expected output:
-// {
-//    femaleUsers: [ 'Tracie May', 'Kendra Hines', 'Rachelle Schneider', 'Lizzie Alford' ],
-//    maleUsers: [ 'Kirby Doyle', 'Kinney Howard', 'Howard Gilmore' ]
-// }
+console.log(genderView(usersArray));
+// // expected output:
+// // {
+// //    femaleUsers: [ 'Tracie May', 'Kendra Hines', 'Rachelle Schneider', 'Lizzie Alford' ],
+// //    maleUsers: [ 'Kirby Doyle', 'Kinney Howard', 'Howard Gilmore' ]
+// // }
 
-// ***************************************************************************
-// Bonus - Iteration 5
-// ***************************************************************************
+// // ***************************************************************************
+// // Bonus - Iteration 5
+// // ***************************************************************************
 
 const data = genderView(usersArray);
 
 const genderCount = data => {
   // Your code goes here ...
```

Comment:
this code is good but since data is an object with two arrays (one for each gender) you could just do a .length and get the amount of elements in each array :) no need for for loops.